test/unit/dialects/postgres/query-generator.test.js

```diff
@@ -557,9 +557,21 @@ if (dialect.match(/^postgres/)) {
         {
           arguments: ['myTable', {name: 'foo'}],
           expectation: "INSERT INTO \"myTable\" (\"name\") VALUES ('foo');"
+        }, {
+          arguments: ['myTable', {name: 'foo'}, {}, { ignoreDuplicates: true }],
+          expectation: {
+            query: 'INSERT INTO "myTable" ("name") VALUES ($1) ON CONFLICT DO NOTHING;',
+            bind: ['foo']
```

Comment:
v4 does not support automatic `bind` with create. You need to rewrite this something like this for v4

```js
arguments: ['myTable', {name: 'foo'}, {}, { ignoreDuplicates: true }],
expectation: 'INSERT INTO "myTable" ("name") VALUES ('foo') ON CONFLICT DO NOTHING;'
```